apps/src/templates/sectionProgress/progressTables/ProgressTableLevelIcon.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {levelTypeWithoutStatus} from '@cdo/apps/templates/progress/progressTypes';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import color from '@cdo/apps/util/color';
+import * as progressStyles from '@cdo/apps/templates/progress/progressStyles';
+import {getIconForLevel} from '@cdo/apps/templates/progress/progressHelpers';
+
+const styles = {
+  container: {
+    ...progressStyles.flex,
+    ...progressStyles.cellContent
+  },
+  icon: {
+    ...progressStyles.inlineBlock,
+    textAlign: 'center',
+    color: color.charcoal,
+    fontSize: 20
+  }
+};
+
+function levelIcon(level) {
```

Comment:
if these are just helper functions to render parts of the component, they should be inside the class. for `sublevelSpace` i think that makes most sense since it doesn't really feel like a useful subcomponent to factor out.

this one though i think makes sense as a subcomponent, in which case this should become`LevelIcon(props)`, and line 58 would become `<LevelIcon icon={getIconForLevel(level, true)} /`>, and line 30 `<FontAwesome icon={props.icon} />